ped/__init__.py

```diff
@@ -147,6 +147,13 @@ def find_file(obj: Any) -> Optional[str]:
                 pass
     except Exception:
         pass
+
+    split_fname = fname.split('/')
```

Comment:
In general, you shouldn't have to worry about path delimiters in Python, since these differ across operating systems (e.g. "/" in *nix "\" in Windows).

Instead prefer using `pathlib` for handling paths.